proposed/http-message.md

```diff
@@ -1161,10 +1161,14 @@ interface UriInterface
      * Implementations SHOULD restrict values to "http", "https", or an empty
      * string but MAY accommodate other schemes if required.
      *
-     * If no scheme is present, this method MUST return an empty string.
+     * The string MUST omit the closing scheme delimiter (":"), if present.
```

Comment:
Question, @Tobion: should I move it there, or remove that verbiage from withQuery()/withFragment() (as it's now in the getQuery()/getFragment() methods)?